src/engraving/style/style.cpp

```diff
@@ -284,6 +284,21 @@ bool MStyle::read(QIODevice* device, bool ign)
     return true;
 }
 
+bool MStyle::isValid(QIODevice* device)
+{
+    XmlReader e(device);
+    while (e.error() == XmlReader::Error::NoError && e.readNextStartElement()) {
+        if (e.name() == "museScore") {
+            while (e.readNextStartElement()) {
+                if (e.name() == "Style") {
+                    return true;
```

Comment:
I'm afraid that this method doesn't guarantee that a style is really valid. A style may contain the "Style" tag, but be corrupted elsewhere

Reply:
It's enough to be confident we can actually extract useful info from it. And it's a significant improvement on MU3 which doesn't even tell you if the file isn't XML at all. Obviously there's always room for improvement but there are far more serious issues to address with very limited time/ resources.  And no spec other than "do what MU3 does".